### PR TITLE
DEVTOOLS-69 If the root raml is deleted, push and push -f fail

### PR DESCRIPTION
**Pull request by fschargorosdky**
* Added the root raml path to the workspace

### CODE REVIEW

File: lib/utils/messages.js [hunk 1]
```diff
@@ -206,16 +206,30 @@ module.exports = function () {
...
+
+      return output;
+    },
+    deletedRootRamlConflict: function (rootRamlPath) {
+      var output = 'The following conflict can\'t be solved with push -f\n\n';
+      var actions = [
+        rootRamlDeletedAction()
+      ];
+      var conflicts = {
+        rootRamlDeleted: rootRamlPath
```
Comment (juancoen) [inline lib/utils/messages.js]: you could define this property as an array to avoid the typeof in buildStatusRespose

File: lib/utils/messages.js [hunk 2]
```diff
@@ -461,11 +475,43 @@ module.exports = function () {
       var message       = action.options.message;
       var colorPrinter  = action.options.color;
 
-      filesChanged.forEach(function (file) {
-        output.push(colorPrinter([prefix, file, message].join(' ')));
-      });
+      if (typeof filesChanged === 'string') {
```
Comment (juancoen) [inline lib/utils/messages.js]: remove this if

File: lib/utils/messages.js [hunk 3]
```diff
@@ -461,11 +475,43 @@ module.exports = function () {
...
 
     return output.join('\n');
   }
+
+  function howToSolveConflicts(conflicts) {
+    var output = 'To resolve run:\n';
+    output += '- ' + TOOL_NAME + ' pull (overwrites local changes)\n';
+    if (conflicts.rootRamlDeleted) {
+      output += 'or\n';
+      output += '- create ' + conflicts.rootRamlDeleted;
```
Comment (juancoen) [inline lib/utils/messages.js]: I don't get this solution. also, if the solution makes sense, it's missing the TOOL_NAME

Comment (juancoen) [inline lib/utils/messages.js]: oh, I got it now, it should be something more descriptive like:
'create a new file and name it: ' + bla

File: lib/utils/messages.js (hunk 1 above)
Comment (fschargorosdky) [inline lib/utils/messages.js]: done

File: lib/utils/messages.js (hunk 2 above)
Comment (fschargorosdky) [inline lib/utils/messages.js]: done

File: lib/utils/messages.js (hunk 3 above)
Comment (fschargorosdky) [inline lib/utils/messages.js]: done

**Comment by juancoen** [comment]
looks good to me

**Pull request merged**